### ISSUE TITLE
[FEATURE REQUEST] VS Code workspace at repository's root level / multi-root workspace

### ISSUE THREAD

**Issue by to-bar** [epiphany-platform/epiphany#2532]
**Is your feature request related to a problem? Please describe.**
VS Code has two kinds of workspaces: `Single-folder` and `Multi-root` (see [docs](https://code.visualstudio.com/docs/editor/workspaces)).
Epiphany currently uses only the `Single-folder` type, which is located in `core\src\epicli` sub-directory. Settings from it are applied to devcontainer and are present after opening this folder. However, Epiphany repo contains also directories and files at the root level (e.g. `docs`, `tools`, changelogs) so developer has a need to work at the root level as well but there is no workspace (settings) defined at that level. VS Code will create `.vscode` directory after changing settings but there is no such folder in Git repo.
As workaround developers have their own settings - so there is no standard rules to follow (e.g. for Markdown extension).

**Describe the solution you'd like**
Introduce `Multi-root` workspace.

> Multi-root workspaces allow you to configure multiple distinct folders to be part of the workspace. Instead of opening a folder as workspace, you will open a <name>.code-workspace JSON file that lists the folders of the workspace. 

**Describe alternatives you've considered**
Add/move workspace at the root level.

**Additional context**
We also need to improve the structure of directories (simplify). This will be done in separate task (after #2533).

---

**DoD checklist**

* [ ] Changelog updated
* [ ] COMPONENTS.md updated / doesn't need to be updated
* [ ] Schema updated / doesn't need to be updated
* [ ] Feature has automated tests
* [ ] Automated tests passed (QA pipelines)
  * [ ] apply
  * [ ] upgrade
* [ ] Idempotency tested
* [ ] Documentation added / updated / doesn't need to be updated
* [ ] All conversations in PR resolved
* [ ] Solution meets requirements and is done according to design doc
* [ ] Usage compliant with license
* [ ] Backport tasks created / doesn't need to be backported

**Comment by seriva** [comment]
I feel this has been all addressed with the major workspace overhaul we did for 1.3 so closing this.
